Sources/SmileID/Classes/SelfieCapture/SelfieViewModel.swift

```diff
@@ -335,7 +335,17 @@ public class SelfieViewModel: ObservableObject, ARKitSmileDelegate {
                     self.error = error
                 }
                 DispatchQueue.main.async { self.processingState = .success }
-            } catch {
+            } catch let error as SmileIDError {
```

Comment:
With this change, we are now no longer handling non-SmileIDErrors. We still would need to handle those